src/feature-flags/flags.ts

```diff
@@ -94,6 +94,28 @@ const InternalFLAGS = {
       visibleInFeatureFlagPanel: true,
     },
   },
+  'pipelineruns-kubearchive': {
```

Comment:
we already have `pipelineruns-kubearchive` and `taskruns-kubearchive` feature flags defined here, so no changes needed in this file